modules/010-deckhouse-flow-schema/templates/flowschema.yaml

```diff
@@ -0,0 +1,191 @@
+# Schema generated by "tools/flow_schema.go" DO NOT EDIT.
```

Comment:
This will not work with external modules.

Reply:
namespace discovery moved to hook